# Request
bool trigger

---
# Response
bool success